app/build.gradle

```diff
@@ -0,0 +1,82 @@
+apply plugin: 'com.android.application'
+
+android {
+    compileSdkVersion 26
+    flavorDimensions "version"
+
+    signingConfigs {
+        releaseconfig {
+            keyAlias 'recipe'
+            keyPassword 'application'
+            storeFile file('/home/shubhank/AndroidStudioProjects/RecipeApplication1/app/build/outputs/apk/Staging/signedapk/staging-release.jks')
```

Comment:
Not accepted. We discussed this many times. Pls do this according to our discussion

Reply:
def Properties props = new Properties()
            def propFile = file('../signing.properties')
            if (propFile.canRead()) {
                props.load(new FileInputStream(propFile))

                if (props != null && props.containsKey('STORE_FILE') && props.containsKey('STORE_PASSWORD') &&
                        props.containsKey('KEY_ALIAS') && props.containsKey('KEY_PASSWORD')) {

                    android.signingConfigs.releaseConfig.storeFile = file(props['STORE_FILE'])
                    android.signingConfigs.releaseConfig.storePassword = props['STORE_PASSWORD']
                    android.signingConfigs.releaseConfig.keyAlias = props['KEY_ALIAS']
                    android.signingConfigs.releaseConfig.keyPassword = props['KEY_PASSWORD']
                } else {
                    android.buildTypes.releaseConfig.signingConfig = null
                }

I changed like this.